example/README.md

```diff
@@ -1,28 +1,45 @@
 # Amplify Flutter Example
 
-Sample application for getting started with the Amplify Flutter Library
-
-## Getting Started
+Sample flutter app for getting started with the Amplify Flutter Library.  This example uses the Auth, Analytics, and Storage components of the Flutter library. 
 
 Please check out our docs here:
 https://docs.amplify.aws/start/q/integration/flutter
 
-!!! IMPORTANT !!!
+# TL;DR 
+
+**The app will not compile.  You must configure 'amplifyconfiguration.dart' using Amplify CLI.**
+
+**Operations such as logging in or uploading images takes time.  Don't click on the app until they complete.**
+
+
+## Getting Started
+
+**IMPORTANT** 
 
-The app will not compile until you have used the Amplify CLI to configure the
+**The app will not compile until you have used the Amplify CLI to configure** the
 AWS resources necessary for running this app.
```

Comment:
Again, we shouldn't suggest they MUST use the CLI (I realize this is what it said before).  Maybe just say '...use the Amplify CLI to configure AWS resources necessary for running this app, or create your own amplifyconfiguration.dart file using the example in our documentation'.

Reply:
thanks added your suggestion